app/src/main/java/com/zulip/android/networking/AsyncGetEvents.java

```diff
@@ -190,7 +190,13 @@ public Void call() throws Exception {
                             } else {
                                 personDao.update(person);
                             }
-                        } catch (Exception e) {
+                            if (person.getEmail().equals(app.getEmail())) {
```

Comment:
Ok, will use `(person.getEmail().equals(app.getYou.getEmail()))` in `narrow_pm_with()`. But I think for future aspect we should update `you` in the `app`. Shall I create different PR for this update part?